### ISSUE TITLE
Should the name "Developer mode" be changed "Monitor mode " or "monitoring mode"?

### ISSUE THREAD

**Issue by sujiniku** [flareteam/flare-engine#1731]
At "Configration" of opening menu, at "Interface" tab,

If "Developer mode", User may misunderstand as programming mode or the mode to design this game.
However, this mode is only displaying the Hero position or some position.

However, this mode may be only to display the Hero position or some position.

https://en.wikipedia.org/wiki/Monitoring

**Comment by dorkster** [comment]
The developer mode option enables using the developer console (press F5 by default when enabled):

![Screenshot_20190830_123152](https://user-images.githubusercontent.com/913982/64037049-a8090200-cb22-11e9-80b0-92be6310830f.png)

The console doesn't provide a way to "program" the game, but it provides many tools useful for developers that are creating a game. Entering `help` in the console will list the available commands. For example, `toggle_devhud` will toggle the visibility of the hero/power position markers.

One of the more powerful commands is `exec`, which takes an [Event](https://github.com/flareteam/flare-engine/wiki/Attribute-Reference#eventmanager) attribute as a parameter. Take for instance this command, which will give the player a health potion:

```
exec reward_item=2,1
```

So in short, I think the Developer Mode is aptly named and doesn't require changing.
